content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,264 @@
+
+If you need assisstance, you can send a message to our **#support-sustaining** slack channel with questions.
+
+
+# Validation
+
+Once installed, the gateway registers itself with Palette. To verify the gateway is registered, navigate to **Tenant Settings > Private Cloud Gateways > MAAS** and verify the gateway is listed on the Manage Private Cloud Gateways page. 
+
+When you install the gateway, an account is auto-created. To verify the account creation, go to **Tenant Settings > Cloud Accounts** and locate **MAAS** in the table. Verify your MAAS account is listed.
+
+
+# Upgrade and Manage the MAAS Gateway
+
+Palette maintains the Operating System (OS) image and all configurations for the PCG. Periodically, the OS images, configurations, and other components need to be upgraded to resolve security or functionality issues. Palette releases upgrades when required, and informs you with an upgrade notification on the gateway.
+
+Review the changes in the upgrade notification, and apply the upgrade when you are ready. 
+
+Upgrading the cloud gateway does not result in any downtime for the tenant clusters. During the upgrade process, new cluster provisioning is unavailable. New cluster requests are queued and processed when the gateway upgrade is complete.
+
+
+# Delete the MAAS Gateway
+
+Follow these steps to delete a MAAS gateway.
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main menu** and select **Tenant Settings > Private Cloud Gateways**.
+
+
+3. Click the **three-dot Menu** for the gateway instance you want to delete and choose **Delete**.
+
+    Palette checks for running tenant clusters associated with the gateway instance and displays an error message if it detects any. 
+    <br />
+
+4. If there are running clusters, delete them and retry deleting the gateway instance.
+
+
+# Resize the MAAS Gateway
+
+You can set up a PCG as a single-node (no HA) or three-node (HA) cluster. You can set up a PCG initially with one node and resize it to three nodes at a later time.
+
+<br />
+
+<InfoBox>
+
+For production environments, we recommend setting up three nodes.
+
+</InfoBox>
+
+
+
+Follow these steps to resize a single-node gateway to three nodes.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the **Main Menu** and select **Tenant Settings > Private Cloud Gateways**.
+
+
+3. Click the **three-dot Menu** for the gateway instance you want to resize and choose **Set number of nodes**.
+
+
+4. Change the number of nodes to 3.
+
+Two new nodes will be created in the cluster.
+
+
+# Next Steps
+
+You can now create tenant clusters in the default cloud account.  To get started, check out [Create and Manage MAAS Clusters](/clusters/data-center/maas/create-manage-maas-clusters).
```

Comment:
I'm not sure what "default" means here for cloud account? seems a remnant

Reply:
Yes, this is a remnant. Changed it to auto-created cloud account.